Makefile.common

```diff
@@ -93,6 +93,7 @@ POLY_BUNDLE=-bundle 'Hacl.Poly1305_32=Hacl.Impl.Poly1305.Field32xN_32' \
   -bundle 'Hacl.Poly1305_128=Hacl.Impl.Poly1305.Field32xN_128' \
   -bundle 'Hacl.Poly1305_256=Hacl.Impl.Poly1305.Field32xN_256'
 NACLBOX_BUNDLE=-bundle Hacl.NaCl=Hacl.Impl.SecretBox,Hacl.Impl.Box
+ECDSA_BUNDLE=-bundle Hacl.Impl.ECDSA=Hacl.Impl.ECDSA,Hacl.Impl.ECDSA.*,Hacl.Impl.P256.*,Hacl.Hash.SHA2,Hacl.Impl.P256,Hacl.Spec.P256.Ladder,Hacl.Impl.SolinasReduction,Hacl.Impl.LowLevel
```

Comment:
For consistency here, I would suggest adding at the end of your bundle argument `[rename=Hacl_ECDSA]`. Alternatively, you could rename the file `Hacl.Impl.ECDSA` into `Hacl.ECDSA`.

This will require updating the `#include` in your test files.

Reply:
Done